honeybadger/contrib/django.py

```diff
@@ -128,3 +129,11 @@ def process_exception(self, request, exception):
         honeybadger.notify(exception)
         clear_request()
         return None
+
+    def __setUserFromContext(self, request):
+        # in Django 1 request.user.is_authenticated is a function, in Django 2+ it's a boolean
+        if hasattr(request, 'user') and (
```

Comment:
Not a blocker, but i think we'll be fine without support for django 1.

Reply:
In our CI, Django 1 is still supported so I thought it's best to support it :)